osu.Game/Overlays/Mods/ModColumn.cs

```diff
@@ -0,0 +1,409 @@
+                                            ScrollbarOverlapsContent = false,
+                                            Child = panelFlow = new FillFlowContainer<ModPanel>
+                                            {
+                                                RelativeSizeAxes = Axes.X,
+                                                AutoSizeAxes = Axes.Y,
+                                                Spacing = new Vector2(0, 7),
+                                                Padding = new MarginPadding(7)
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            createHeaderText();
+
+            if (allowBulkSelection)
+            {
+                controlContainer.Height = 35;
+                controlContainer.Add(toggleAllCheckbox = new ToggleAllCheckbox(this)
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    Scale = new Vector2(0.8f),
+                    RelativeSizeAxes = Axes.X,
+                    LabelText = "Enable All",
+                    Shear = new Vector2(-ModPanel.SHEAR_X, 0)
+                });
+                panelFlow.Padding = new MarginPadding
+                {
+                    Top = 0,
+                    Bottom = 7,
+                    Horizontal = 7
+                };
+            }
+        }
+
+        private void createHeaderText()
+        {
+            IEnumerable<string> headerTextWords = modType.Humanize(LetterCasing.Title).Split(' ');
+
+            if (headerTextWords.Count() > 1)
+            {
+                headerText.AddText($"{headerTextWords.First()} ", t => t.Font = t.Font.With(weight: FontWeight.SemiBold));
+                headerTextWords = headerTextWords.Skip(1);
+            }
+
+            headerText.AddText(string.Join(' ', headerTextWords));
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuGameBase game, OverlayColourProvider colourProvider, OsuColour colours)
+        {
+            availableMods.BindTo(game.AvailableMods);
+
+            headerBackground.Colour = accentColour = colours.ForModType(modType);
+
+            if (toggleAllCheckbox != null)
+            {
+                toggleAllCheckbox.AccentColour = accentColour;
+                toggleAllCheckbox.AccentHoverColour = accentColour.Lighten(0.3f);
+            }
+
+            contentContainer.BorderColour = ColourInfo.GradientVertical(colourProvider.Background4, colourProvider.Background3);
+            contentBackground.Colour = colourProvider.Background4;
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            availableMods.BindValueChanged(_ => Scheduler.AddOnce(updateMods), true);
```

Comment:
Unsure about this, is going to force a one-frame delay after `LoadComplete` before the async load of the panels starts. Maybe best to move the initial call out to a separate line?